### ISSUE TITLE
NestJS Swagger crashed when using a controller with an HTTP method decorator that has a string array as the path parameter

### ISSUE THREAD

**Issue by JasonHK** [nestjs/swagger#268]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

NestJS Swagger crashed when using a controller with an HTTP method decorator (like `@Get()`) that has a string array as the path parameter.

## Expected behavior

The core NestJS framework actually works flawlessly and can resolve the path. The only part not work is NestJS Swagger.

## Minimal reproduction of the problem with instructions
```ts
@Controller()
export class ARandomController {

  @Get(["/", "/hot", "/popular"])
  @UseInterceptors(FeedSerializerInterceptor)
  public async getSomething(): Promise<Feed> { ... }
}
```

## Environment

<pre><code>
NestJS version: 6.3.1

 
For Tooling issues:
- Node version: 10.16.0  
- Platform: Linux 

Others:

- NestJS Swagger version: 3.0.2
</code></pre>

**Comment by diephil** [comment]
Hey @JasonHK ! Did you find any solution for this ?

**Comment by victordidenko** [comment]
Put exception example, because I didn't find this issue (and wanted to create new one) by string `str.substr is not a function`
```
TypeError: str.substr is not a function
    at Function.parse (.../node_modules/path-to-regexp/index.js:107:28)
    at SwaggerExplorer.validateRoutePath (.../node_modules/@nestjs/swagger/dist/swagger-explorer.js:107:41)
    at SwaggerExplorer.exploreRoutePathAndMethod (.../node_modules/@nestjs/swagger/dist/swagger-explorer.js:93:44)
    at explorers.reduce (.../node_modules/@nestjs/swagger/dist/swagger-explorer.js:55:45)
    at Array.reduce (<anonymous>)
    at lodash_1.mapValues (.../node_modules/@nestjs/swagger/dist/swagger-explorer.js:54:97)
    at .../node_modules/lodash/lodash.js:13401:38
    at .../node_modules/lodash/lodash.js:4905:15
    at baseForOwn (.../node_modules/lodash/lodash.js:2990:24)
    at Function.mapValues (.../node_modules/lodash/lodash.js:13400:7)
```

**Comment by kamilmysliwiec** [comment]
For now, we'll simply grab only the first element in the array since there are no *aliases* feature in OAS3 specification. 

Fixed in 4.0.0
